src/index.html

```diff
@@ -1,13 +1,298 @@
+              <a href="tel:+1 234 555-55-55" class="icon icon--phone">
+                <div class="icon--phone-hint">+1 234 555-55-55</div>
+              </a>
+              <a href="#menu" class="icon icon--menu"></a>
+            </div>
+          </div>
+
+          <div class="header__bottom">
+            <h1 class="header__title">Take the Streets</h1>
+          </div>
+        </div>
+      </div>
+    </header>
+
+    <aside class="page__menu menu" id="menu">
+      <div class="container">
+        <div class="top-bar menu__top">
+          <a href="#" class="top-bar__logo-link">
+            <img
+              class="top-bar__logo"
+              src="images/logo-menu-2px.png"
+              alt="Bike logo"
+            >
+          </a>
+
+          <a href="#" class="icon icon--close"></a>
+        </div>
+
+        <div class="menu__bottom">
+          <nav class="nav menu__nav">
+            <ul class="nav__list">
+              <li class="nav__item">
+                <a class="nav__link" href="#">Home</a>
+              </li>
+              <li class="nav__item">
+                <a class="nav__link" href="#about-us">About us</a>
+              </li>
+              <li class="nav__item">
+                <a class="nav__link" href="#compare-bikes">Compare bikes</a>
+              </li>
+              <li class="nav__item">
+                <a class="nav__link" href="#details">Details</a>
+              </li>
+              <li class="nav__item">
+                <a class="nav__link" href="#contacts">Contacts</a>
+              </li>
+            </ul>
+          </nav>
+
+          <a href="tel:+1 234 5555-55-55" class="menu__phone-number">
+            +1 234 5555-55-55
+          </a>
+
+          <a href="tel:+1 234 5555-55-55" class="menu__book-a-test-ride">
+            Book a test ride
+          </a>
+        </div>
+      </div>
+    </aside>
+
+    <main class="main">
+      <div class="container">
+        <div class="main__content">
+          <section class="about-us" id="about-us">
+            <h2 class="
+              about-us__title
+              section-title
+              section-title--align--left
+              ">
+                Move Free
+            </h2>
+
+            <div class="about-us__content">
+              <p class="about-us__description">Shift your ride, not gears. Find the fastest way to move in the city as the bike adapts intuitively to power the speed you need.</p>
+            </div>
+          </section>
+
+          <section class="compare-bikes" id="compare-bikes">
+            <h2 class="section-title">Compare Bikes</h2>
+
+            <div class="recommended__products">
+              <article class="product recommended__product">
+                <a href="#" class="product__link">
+                  <img
+                  src="images/products/Sporty 4.png"
+                  alt="Sporty 4"
+                  class="product__photo"
+                  >
```

Comment:
Need to check and fix it everywhere
```suggestion
                  <img
                    src="images/products/Sporty 4.png"
                    alt="Sporty 4"
                    class="product__photo"
                  >
```